blockchain/chain_test.go

```diff
@@ -116,15 +120,10 @@ func TestHaveBlock(t *testing.T) {
 // combinations of inputs to the CalcSequenceLock function in order to ensure
 // the returned SequenceLocks are correct for each test instance.
 func TestCalcSequenceLock(t *testing.T) {
-	fileName := "blk_0_to_4.dat.bz2"
-	blocks, err := loadBlocks(fileName)
-	if err != nil {
-		t.Errorf("Error loading file: %v\n", err)
-		return
-	}
+	netParams := &chaincfg.SimNetParams
 
 	// Create a new database and chain instance to run tests against.
-	chain, teardownFunc, err := chainSetup("haveblock", &chaincfg.MainNetParams)
+	chain, teardownFunc, err := chainSetup("haveblock", netParams)
```

Comment:
Haven't seen any signs of negative interactions yet myself, but changing to `calcseqlock` just to be safe :)